src/optimisation/Optimisation.jl

```diff
@@ -273,6 +273,35 @@ StatsBase.params(m::ModeResult) = StatsBase.coefnames(m)
 StatsBase.vcov(m::ModeResult) = inv(StatsBase.informationmatrix(m))
 StatsBase.loglikelihood(m::ModeResult) = m.lp
 
+"""
+    Base.get(m::ModeResult, var_symbol::Symbol)
+    Base.get(m::ModeResult, var_symbols::Vector{Symbol})
+
+Return the values of all the variables with the symbol(s) `var_symbol` in the mode result
+`m`. The return value is a `NamedTuple` with `var_symbols` as the key(s).
```

Comment:
Intuitively I would assume that indexing with a `Tuple` might (possibly) return a `NamedTuple`. But returning a `NamedTuple` for `Vector` inputs seems quite surprising, and also means that not even the keys of the `NamedTuple` return type can be inferred.

Reply:
Good point. I changed this so that the second argument can be any iterator type, and made the single symbol version use a singleton tuple. I also stabilised the array types a bit. Let me know if you are happy with this.